src/js/main.js

```diff
@@ -699,6 +701,78 @@ function generateFilename(prefix, suffix) {
     return `${filename}.${suffix}`;
 }
 
+/*
+    * NWjs File Dialog implementation. Probably needs it's own module.
```

Comment:
Sure. Better to have our own module/file with a wrapper over fs if we are not using it directly in code.

Reply:
Already tried to put it module/file but having problems with import / export and also require.
The build is failing now for Android and Windows 😕 I can't catch an artifact to test with 😞 
I'll hope Android is not going to be a deal breaker.

Reply:
Using same strategy as common.js almost succeeded, ended up putting the wrapper in common.js
Now only Android build is failing 😏

Reply:
This can be acceptable, but, can't have it's own file? Something like `file_util.js` or `file_common.js` or similar? I think is better to have more `.js` files but small. Now we have some of them very, very big.